src/utils/NodeList.js

```diff
@@ -272,7 +272,6 @@ class NodeList {
         })
       }
       document.addEventListener('click', blurEvent, false)
-      document.addEventListener('touchstart', blurEvent, false)
```

Comment:
This onBlur is an util function, and is probably used in other components as well. I felt if we change it here we might cause some unexpected behaviour from other components.

If we just want to disable it in Navbar.vue, I would suggest add a boolean flag to the `onBlur (callback)` method to control if we want trigger by `touchstart` event, and default value is true: `onBlur (callback, enableTriggerByTouchStart = true)`. Do a flag check before we add event listener in the method.

After that, in the NavBar, we can change it to 
```
.onBlur((e) => {
  $dropdown.each((content) => {
    if (!content.contains(e.target)) content.classList.remove('open')
  })
}, /*enableTriggerByTouchStart=*/ false)
```

Reply:
Good point, I've done exactly that. However, it wasn't the onBlur function in Navbar.vue that had to be changed, but rather the one in Dropdown.vue. 

If I'm understanding the code correctly, the `onBlur` in Dropdown.vue will be called when a navbar item is out of focus, and since the `enableTriggerByTouchStart` is false, only the dropdown closes rather than the the entire navbar.